RELEASE.md

```diff
@@ -1,9 +1,13 @@
 # Release 0.2.0
+- This release includes the required ontology ttl files in the release itself.
+- Removed test data from release package to save space.
+
+# Release 0.2.0 (no longer available)
```

Comment:
You've duplicated the version number here.

Also, why no longer available? Even if it isn't on PyPI, I wouldn't state this here, as people can get it from GitHub!